cms/djangoapps/contentstore/management/commands/backfill_orgs_and_org_courses.py

```diff
@@ -82,7 +82,7 @@ def handle(self, *args, **options):
             ({"short_name": orgslug}, courseid)
             for orgslug, courseid in sorted(orgslug_courseid_pairs)
         ]
-        if not confirm_changes(options, orgs, orgslug_courseid_pairs):
+        if not confirm_changes(options, orgs, org_courseid_pairs):
```

Comment:
`orgslug_courseid_pairs` is in fact a variable, but it's of the type `list[tuple[str, str]]`.

We wanted to pass in `org_courseid_pairs`, which is of type `list[tuple[dict, str]]`.